xml/pop_banners.xml

```diff
@@ -217,7 +217,58 @@ matched against these patterns to fingerprint POP3 servers.
     <param pos="0" name="service.product" value="Courier POP"/>
   </fingerprint>
   <!--
-
+<fingerprint pattern= "^CMailServer ([\d\.]+) POP3 Service Ready$">
+    <description>matches to CMailServer</description>
+    <example>CMailServer 5.0.0 POP3 Service Ready</example>
```

Comment:
Where possible, examples should include tests for values that are extracted. This ensures that recog is capturing the values that you expect.
In this case it would be:
`<example service.version="5.0.0">CMailServer 5.0.0 POP3 Service Ready</example>`